tests/pytests/test_erisieve.py

```diff
@@ -174,39 +174,41 @@ def test_schwarz_vs_density_quartets():
         no_com
     """)
 
-    # Run a quick SCF job to get density matrix
-    energy, wfn = psi4.energy('hf/DZ', return_wfn=True)
-
-    psi4.set_options({"ints_tolerance" : 1e-12})
-    basis = wfn.basisset()
-
-    factory = psi4.core.IntegralFactory(basis)
-    psi4.set_options({ "screening" : 'schwarz', 'integral_package': 'libint2' })
-    eriSchwarz = factory.eri(0)
-
-    psi4.set_options({"screening" : 'density' })
-    eriDensity = factory.eri(0)
-    Da = [wfn.Da()]
-    # Update using the Density Matrix
-    eriDensity.update_density(Da)
-
-    shell_inds = range(basis.nshell())
-    quartets = itertools.product(shell_inds, shell_inds, shell_inds, shell_inds)
-
-    screen_count_schwarz = 0
-    screen_count_density = 0
-
-    for m, n, r, s in quartets:
-        screen_schwarz = not eriSchwarz.shell_significant(m, n, r, s)
-        screen_density = not eriDensity.shell_significant(m, n, r, s)
-
-        if screen_schwarz:
-            screen_count_schwarz += 1
-        if screen_density:
-            screen_count_density += 1
-    
-    assert compare_integers(screen_count_schwarz, 14504, 'Schwarz Screened Ints Count, Cutoff 1.0e-12')
-    assert compare_integers(screen_count_density, 19440, 'Density Screened Ints Count, Cutoff 1.0e-12')
+    # run schwarz screening calculation
+    psi4.set_options({ 
+        "scf_type": "direct", 
+        "screening" : 'schwarz', 
+        "df_scf_guess" : False,
+        "integral_package": 'libint2', 
+        "ints_tolerance" : 1e-12, 
+        "save_jk": True,
+        "bench" : 1 
+
+    })
+    schwarz_energy, schwarz_wfn = psi4.energy('hf/DZ', return_wfn=True)
+
+    # run density screening calculation
+    psi4.set_options({ 
+        "scf_type": "direct", 
+        "screening" : 'density', 
+        "df_scf_guess" : False,
+        "integral_package": 'libint2', 
+        "ints_tolerance" : 1e-12,
+        "save_jk": True,
+        "bench" : 1
+    })
```

Comment:
Unless a `psi4.core.clear_options()` has been run in-between, options persist, so only `{"screening": "density"}` technically needed. 

No need for changes -- this is nice and clear -- just FYI.

Reply:
Hmm, this is good to know. Thanks for informing me!